open_alchemy/helpers/peek.py

```diff
@@ -855,9 +858,11 @@ def _peek_key(
     _check_schema_schemas_dict(schema, schemas)
 
     # Base case, look for type key
-    value = schema.get(key)
-    if value is not None:
-        return value
+    keys = [key.replace("x-", prefix) for prefix in VALID_PREFIXES]
+    for k in keys:
```

Comment:
There is a one liner that could be used here, let me see if I can manage it
```python
value = first(filter(lambda value: value is not None, map(schema.get, keys)))
```

Reply:
Nice! It works well!

I tend to use short variables, for indexes or sort of temporary variables, but I can definitely make the effort of avoiding them if you'd prefer.